semantic_search/__init__.py

```diff
@@ -1,7 +1,7 @@
 import sys
 sys.path.append('../..')
 from aips import get_entity_extractor, get_sparse_semantic_search
-from .query_tree import enrich, to_query_string
```

Comment:
add import `to_queries` (which replaced/should replace `_to_query_string`).

Reply:
Done 👍